launch_testing_ros/test/examples/talker_listener_launch_test.py

```diff
@@ -32,16 +34,18 @@ def generate_test_description(ready_fn):
     # 'chatter' topic, but we want to show how to use remappings to munge the data so we
     # will remap these topics when we launch the nodes and insert our own node that can
     # change the data as it passes through
+    path_to_test = os.path.dirname(__file__)
```

Comment:
Why moving the `examples` folder into `test`?
IMO, examples should be at the top level dir of the package, where they are easier to find.

Reply:
Same as above.